freqtrade/freqai/data_kitchen.py

```diff
@@ -331,11 +344,17 @@ def normalize_data_from_metadata(self, df: DataFrame) -> DataFrame:
         :param df: Dataframe to be standardized
         """
 
+        if not [col for col in df.columns if col.startswith('PC')]:
```

Comment:
If we are lucky, even this function will remain entirely unchanged since `item` is unique. See above comments and please confirm my suspicion.